lib/galaxy/tool_util/linters/inputs.py

```diff
@@ -115,10 +115,14 @@
 ]
 
 
-def lint_inputs(tool_xml, lint_ctx):
+def lint_inputs(tool_source, lint_ctx):
```

Comment:
A bunch of unit tests that call this directly fail because of the input type change.

```suggestion
def lint_inputs(tool_source: XmlToolSource, lint_ctx: LintContext):
```